.gitignore

```diff
@@ -11,6 +11,8 @@ package/dist/**
 /tests/cov_profile
 /tests/cov_profile.lcov
 /tests/test-out.json
+/tests/docs/playwright
```

Comment:
This will likely not be merged, yet I need to know what to do with the artifacts that are created when I run tests.